### PR TITLE
[tracker-miners] Update to 3.2.1. JB#55697 OMP#JOLLA-414

### PR DESCRIPTION
**Pull request by pvuorela**
Most importantly fixes duplicate entries created to content specific
graphs in case tracker handles new files when they are empty.

Couple patches dropped as they got merged upstream.

@Tomin1 @spiiroin @aroditelev

### CODE REVIEW

File: rpm/0001-Tracker-config-overrides.patch [hunk 1]
```diff
@@ -1,7 +1,7 @@
-From aae99517e9159f9eb7980fb1e7e53606c8b8a04f Mon Sep 17 00:00:00 2001
+From c48ac84bc9c8b591d99fb1091e58dccc0d7fa710 Mon Sep 17 00:00:00 2001
 From: Islam Amer <islam.amer@jollamobile.com>
 Date: Wed, 12 Nov 2014 18:10:53 +0200
-Subject: [PATCH 1/7] Tracker config overrides
+Subject: [PATCH 1/5] Tracker config overrides
```
Comment (rainemak) [inline rpm/0001-Tracker-config-overrides.patch]: It's a good habit to use --no-numbered (-N) when formatting patches. Similarly --zero-commit is good habit.

File: rpm/0002-Fix-systemd-unit-files.patch [hunk 2]
```diff
@@ -1,7 +1,7 @@
-From 2032310bf3bcfe3eb2822dab1a1a22b8fc234add Mon Sep 17 00:00:00 2001
+From e5926d7382bacc446bbd8d7338e31036c3e1436c Mon Sep 17 00:00:00 2001
```
Comment (rainemak) [inline rpm/0002-Fix-systemd-unit-files.patch]: --zero-commit when formatting.

**Comment by okodron** [comment]
Should we update tracker as well to keep these packages in sync?

**Comment by pvuorela** [comment]
> Should we update tracker as well to keep these packages in sync?

Yea, could make sense now that the 3.2.1 was just released on that side too and it's not too big of an update anyway.

File: rpm/0001-Tracker-config-overrides.patch (hunk 1 above)
Comment (pvuorela) [reply rpm/0001-Tracker-config-overrides.patch]: Sure. Regenerated the patches.

**Review by mlehtima — APPROVED** [review]

**Review by Tomin1 — APPROVED** [review]

File: rpm/tracker-miners.spec [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Name:       tracker-miners
 Summary:    Tracker miners and metadata extractors
-Version:    3.2.0
+Version:    3.2.1
 Release:    1
 License:    LGPLv2+ and GPLv2+
 URL:        https://wiki.gnome.org/Projects/Tracker
```
Comment (okodron) [inline rpm/tracker-miners.spec]:
```suggestion
URL:        https://gnome.pages.gitlab.gnome.org/tracker/
```

**Review by okodron — APPROVED** [review]

File: rpm/tracker-miners.spec (hunk 3 above)
Comment (pvuorela) [reply rpm/tracker-miners.spec]: Done.

**Review by xlise — APPROVED** [review]

**Pull request merged**